### PR TITLE
Ref/pn 9170

### PR DESCRIPTION
**Pull request by leleOFA**
## List of changes proposed in this pull request

- fix previous pr

## How to test

 ./mvnw clean test -Dcucumber.filter.tags=@TA_PGaccettaDelegaConGruppoConErrore

- [ ] Does this PR require a manual test in AWS CodeBuild?

### CODE REVIEW

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java [hunk 1]
```diff
@@ -308,11 +288,12 @@ public void siControllaCheLaDelegaPGALoStatoAttiva(String ragioneSociale) {
...
-    @And("Non si assegna un gruppo alla delega")
-    public void nonSiAssegnaUnGruppoAllaDelega() {
-        logger.info("Si clicca sul bottone non assegna gruppo");
+    @And("Si clicca sul bottone conferma gruppo errato")
+    public void siCliccaSulBottoneConfermaGruppoErrato() {
+        logger.info("Si seleziona il bottone conferma");
 
-        deleghePGPagoPAPage.clickNonAssegnaGruppo();
+        deleghePGPagoPAPage.clickBottoneConferma();
+        deleghePGPagoPAPage.verificaEsistenzaErroreCodiceSbagliato();
```
Comment (afus0) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java:296]: You need to verify if the value inside is 'Codice di verifica non valido' and then check if the input boxes of the OTP are marked red

File: src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/DeleghePGPagoPAPage.java [hunk 2]
```diff
@@ -383,6 +388,29 @@ public void checkTextboxCodiceSonoRosse() {
         }
     }
 
+    public void checkErroreInSelezioneGruppo() {
+        gruppoField.sendKeys("Errore Test");
```
Comment (afus0) [inline src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/DeleghePGPagoPAPage.java:396]: what's the purpose of this action?

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java [hunk 3]
```diff
@@ -924,6 +924,8 @@ public void siCliccaSullaSecondaPaginaDelleNotifiche(int pagina) {
     }
 
 
+
+
```
Comment (afus0) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java]: format the code and remove extra spaces

**Review by afus0 — CHANGES_REQUESTED** [review]

File: src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/DeleghePGPagoPAPage.java (hunk 2 above)
Comment (leleOFA) [reply src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/DeleghePGPagoPAPage.java:396]: insert group and check it not exist

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java (hunk 3 above)
Comment (leleOFA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java]: delete

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java (hunk 1 above)
Comment (leleOFA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java:296]: done

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java [hunk 4]
```diff
@@ -924,9 +924,7 @@ public void siCliccaSullaSecondaPaginaDelleNotifiche(int pagina) {
     }
 
 
-    /**
-     * A simple object that represents the esito notifica, i.e. the return value of siVerificaEsitoNotifica.
-     */
+
```
Comment (afus0) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java]:
```suggestion
/**
 * A simple object that represents the esito notifica, i.e. the return value of siVerificaEsitoNotifica.
 */
```

**Review by afus0 — CHANGES_REQUESTED** [review]

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java (hunk 4 above)
Comment (leleOFA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/mittente/NotificaMittentePagoPATest.java]: done

**Review by afus0 — APPROVED** [review]

**Pull request merged**